src/group/libp2p_group_gossip_server.erl

```diff
@@ -44,6 +44,7 @@
 -define(DNS_TIMEOUT, 2000). % millis
 -define(DNS_SLEEP, 100). % millis
 -define(DEFAULT_SEED_DNS_PORTS, [2154, 443]).
+-define(SEED_CONFIG_DNS_NAME, "_seed_config.helium.io").
```

Comment:
the DNS record is configutable, can we make this configurable too?

Reply:
we don't seem to use this anymore now that we pull it from the app env.  should it be the default there, or is there some other place that it's used?